module/mempool/herocache/execution_data.go

```diff
@@ -0,0 +1,97 @@
+package herocache
+
+import (
+	"fmt"
+
+	"github.com/rs/zerolog"
+
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/module"
+	"github.com/onflow/flow-go/module/executiondatasync/execution_data"
+	herocache "github.com/onflow/flow-go/module/mempool/herocache/backdata"
+	"github.com/onflow/flow-go/module/mempool/herocache/backdata/heropool"
+	"github.com/onflow/flow-go/module/mempool/herocache/internal"
+	"github.com/onflow/flow-go/module/mempool/stdmap"
+)
+
+type BlockExecutionData struct {
+	c *stdmap.Backend
+}
+
+// NewBlockExecutionData implements a block execution data mempool based on hero cache.
+func NewBlockExecutionData(limit uint32, logger zerolog.Logger, collector module.HeroCacheMetrics) *BlockExecutionData {
+	t := &BlockExecutionData{
```

Comment:
It can be `return`ed directly without assigning to a variable.

Reply:
[bdaa257](https://github.com/onflow/flow-go/pull/4197/commits/bdaa257c41061d70489910a54f45d61d62fa17b8)